src/yunohost/app.py

```diff
@@ -156,11 +156,24 @@ def app_info(app, full=False):
 
 
 def _app_upgradable(app_infos):
+    from packaging import version
```

Comment:
Well in fact ... not sure what's happening on my LXC but `python-packaging` is not installed despite `python-pkg-resources` being installed ... Anyway it's good to explicitly state dependencies in `debian/control` even though it implicitly depends on other dependency X just in case we drop X for some reason...

Reply:
Well I see your commit https://github.com/YunoHost/yunohost/pull/864/commits/5c6b4118b7d41f2fb2b25034eb92fb825d4d255c but let me reiterate ... 
- I'm trying to `import packaging` on my RPi2 which runs yunohost, and it does **not** works even though `python-pkg-resources` is installed...
- Also, `apt show python-jinja2` will show that `python-pkg-resources` is **not** a Depends of `python-jinja2` but only a Recommends, so you have no absolute guarantee that it's gonna be installed
- `python-pkg-resources` does **not** depends either on `python-packaging` as a deb package but it apparently ships packaging in `/usr/lib/python2.7/dist-packages/pkg_resources/_vendor/` but that's not loaded with the default `$PYTHONPATH`

TL;DR : let's just add `python-packaging` to the damn debian/control because anyway, it's free, and it's more robust and transparent to have an explicit dependency relationship than an implicit one ...

Reply:
Sorry I didn't really understood what you want ? So now it should be better.